isis/tests/Fixtures.cpp

```diff
@@ -321,11 +321,16 @@ namespace Isis {
       cubeL = new Cube();
       cubeR = new Cube();
 
-      cubeLPath = tempDir.path() + "/observationPairL.cub";
+      cubeLPath = "/Users/acpaquette/Desktop/observationPairL.cub";
```

Comment:
I think you forgot to change the path back.

Reply:
Every darn time. Thanks @kaitlyndlee 👍